experimental/dds/tree2/src/core/undo/undoRedoManager.ts

```diff
@@ -181,36 +179,30 @@ export class UndoRedoManager<TChange, TEditor extends ChangeFamilyEditor> {
 
 	/**
 	 * Updates the state of this {@link UndoRedoManager} to correctly reference commits that have been rebased after merging.
-	 * @param newHead - the head commit of the newly rebased branch.
+	 * @param newCommits - All commits which were not part of the source branch before the rebase, but are after the rebase.
```

Comment:
This was the doc comment of `newSourceCommits` on `rebaseBranch` _before_ we changed the meaning, so, maybe this language is incorrect or at least ambiguous. It's more like "all commits which were appended to the source branch"